guides/source/error_reporting.md

```diff
@@ -94,7 +103,7 @@ end
 
 #### Reporting and Re-raising Errors
 
-[`Rails.error.record`](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html#method-i-record) will report errors to all registered subscribers and then re-raise the error, meaning that the rest of your code won't execute.
+[`Rails.error.record`](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html#method-i-record) will report errors to all registered subscribers and then **re-raise** the error, meaning that the rest of your code won't execute.
```

Comment:
We tend not to start a sentence with link text. Same comment for the other methods in the section.

```suggestion
The [`Rails.error.record`](https://api.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html#method-i-record) method will report errors to all registered subscribers and then **re-raise** the error, meaning that the rest of your code won't execute.
```